src/cpp/r/R/Tools.R

```diff
@@ -414,6 +414,13 @@ assign(envir = .rs.Env, ".rs.getVar", function(name)
       unlockBinding(name, packageEnv)
       assign(name, new, packageName)
       lockBinding(name, packageEnv)
+
+      if (namespace && package %in% loadedNamespaces()) {
+         ns <- asNamespace(package)
```

Comment:
@jmcphers  I see. Yes, I guess a function could be defined in the package but removed from the environment, correct? PR updated.